MediaPlayer/tests/util-specs.js

```diff
@@ -0,0 +1,72 @@
+import {memoize} from '@enact/core/util';
+import DurationFmt from 'ilib/lib/DurationFmt';
+
+import {parseTime, secondsToTime} from '../util';
```

Comment:
Could we test `secondsToPeriod` and `countReactChildren` here too? Even though it seems been covered by other tests.

Reply:
Added the case.